node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/configserver/reports/BaseReport.java

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.hosted.node.admin.configserver.reports;
```

Comment:
Shouldn't this be under `com.yahoo.vespa.hosted.node.admin.configserver.noderepository.bindings`?

Reply:
Moved to under noderepository.